onnxruntime/core/mlas/lib/convolve.cpp

```diff
@@ -956,6 +960,35 @@ Return Value:
 
     const MLAS_CONV_ALGORITHM Algorithm = Parameters->Algorithm;
 
+#if defined(USE_KLEIDIAI) && !defined(_MSC_VER)
+    if (Algorithm == MlasConvAlgorithmExpandThenGemmSegmented &&
```

Comment:
Maybe we can just use MlasConvAlgorithmDepthwise ? Please see https://github.com/microsoft/onnxruntime/pull/26688 ?